tests/components/mobile_app/conftest.py

```diff
@@ -63,3 +63,64 @@ def registry(hass):
     """Configure the websocket_api component."""
     assert await async_setup_component(hass, "websocket_api", {})
     await hass.async_block_till_done()
+
+
+@pytest.fixture
+async def encrypt_payload():
+    """Return a encrypted payload given a key and dictionary of data."""
+
+    def do_encrypt(secret_key, payload):
```

Comment:
Requesting specific attention here and at line 100. This was the [StackOverflow approved method](https://stackoverflow.com/a/28570677/486182) of passing in parameters to a pytest fixture but it feels... not great. Should this not be a fixture? Only downside to that would be losing `pytest.skip` if we try to import NaCL and fail.

Reply:
Don't make it a ficture and just move the `do_encrypt` function out if it and use that, that should work ?